lib/taurus/qt/qtgui/display/tauruslabel.py

```diff
@@ -231,6 +231,7 @@ class TaurusLabel(Qt.QLabel, TaurusBaseWidget):
     def __init__(self, parent=None, designMode=False):
         self._prefix = self.DefaultPrefix
         self._suffix = self.DefaultSuffix
+        self._permanentText = None
```

Comment:
This new property should be registered as a configproperty

Reply:
Done